app/services/nomis_alerts/importer.rb

```diff
@@ -126,8 +126,9 @@ class Importer
       'Health and medical' => :health_issue
     }.freeze
 
-    def initialize(alert_types:, alert_codes:)
-      self.alert_types = alert_types.map { |alert_type| [alert_type[:code], alert_type] }.to_h.with_indifferent_access
+    def initialize(alert_codes:)
+      self.alert_types =
```

Comment:
Sorry to be nitpicky. This doesn't need to go in the initialiser, it could be lazily evaluated in a `alert_types` method. For example:

```ruby
def alert_types
  @alert_types ||= NomisClient::AlertTypes.get.map { |alert_type| [alert_type[:code], alert_type] }.to_h.with_indifferent_access
end
```
Not a blocker however!

Reply:
Even better, you could also consider some convenience method on `NomisClient::AlertTypes` to return it in the format you want.